service/src/main/resources/changelog/changelog.yaml

```diff
@@ -89,6 +89,9 @@ databaseChangeLog:
   - include:
       file: changesets/20231102_set_table_role.yaml
       relativeToChangelogFile: true
```

Comment:
The set_table_role changeset is the actual changeset that needs to be at the end. It looks like _someone_ (oops!) forgot to honor this the last time the file was updated. So the order should go:
- add_workspace_id
- github_method_details
- set_table_role

And I wonder whether we can make the reminder more easy to notice somehow 🤔

Reply:
I added the name of the change set in the reminder to maybe make it easier to notice?